app/javascript/app/components/sankey/sankey-node/sankey-node-styles.scss

```diff
@@ -0,0 +1,8 @@
+@import '~styles/layout.scss';
+
+.nodeText {
+  font-size: $font-size-s;
+  font-weight: $font-weight-light;
+  font-family: $font-family-1;
```

Comment:
Maybe is not related to this PR and could be done in an independent one but I've noticed that we are not using the proper font family on visualizations (charts and maps) tooltips. 
We are using `Lato` instead of `Roboto Mono` as proposed in the [Zeplin](https://zpl.io/aX450PP) designs.
We should add `@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');` to the `base.scss` file and add another `$font-family` variable to the `settings.scss`.
Let me know if you want me to take care of this in another PR.